plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -396,9 +396,11 @@ def __init__(self, step_type, downloads=None, repo=None, conduit=None, config=No
 
         # Even if basic auth is enabled, it should only be used for the token requests which are
```

Comment:
i guess we can remove this comment since it is not the case anymore?

Reply:
actually no, we need to update this that the auth credentials will be used based on the scheme returned in the 401 response headers